### PR TITLE
fix #236

### LINKED ISSUE #236: Build artifacts when a new PR is posted
This can potentially be done using GitHub services, or if needed the small server that we use to host documentation.

- Pull down the branch
- Run make docs
- Post a link on the PR to the live artifacts.

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [artifact 1](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempress-tree.qzv)

File: .travis.yml [hunk 1]
```diff
@@ -31,3 +31,5 @@ script:
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza docs/moving-pictures/unweighted_unifrac_pcoa_results.qza --filter-extra-samples
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza docs/moving-pictures/biplot.qza --filter-extra-samples
+  # now we can upload the files to mchelper
```
Comment (ElDeveloper) [inline .travis.yml:34]: Can you add this under the `after_success:` directive so we don't build "failed" PRs?

File: .travis.yml [hunk 2]
```diff
@@ -31,3 +31,5 @@ script:
...
+  - curl -POST -F files[]=@"${PWD}/docs/moving-pictures/empress-tree.qzv" files[]=@"${PWD}/docs/moving-pictures/empress-tree-tandem.qzv" files[]=@"${PWD}/docs/moving-pictures/empress-tree-tandem-biplot.qzv" http://mchelper.ucsd.edu:8888/uploads/empress/${TRAVIS_PULL_REQUEST}/
```
Comment (ElDeveloper) [inline .travis.yml:35]: When this is run from the master branch i.e. once things are merged, we'll need to adjust the value to use "master" when the value of this environment variable is `false`.

The docs say:

    TRAVIS_PULL_REQUEST: The pull request number if the current job is a pull request, “false” if it’s not a pull request.

Can you add an If statement to account for that?

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempress-tree.qzv)

File: .travis.yml [hunk 3]
```diff
@@ -31,3 +31,8 @@ script:
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza docs/moving-pictures/unweighted_unifrac_pcoa_results.qza --filter-extra-samples
   - ./tests/python/make-dev-page.py docs/moving-pictures/rooted-tree.qza docs/moving-pictures/table.qza docs/moving-pictures/sample_metadata.tsv docs/moving-pictures/taxonomy.qza docs/moving-pictures/biplot.qza --filter-extra-samples
+after_success:
+  - if [ ${TRAVIS_PULL_REQUEST} == "false" ]; then export TRAVIS_PULL_REQUEST=${TRAVIS_BRANCH} ; fi
+  - echo ${TRAVIS_PULL_REQUEST}
+  # now we can upload the files to mchelper
+  - curl -POST -F files[]=@"${PWD}/docs/moving-pictures/empress-tree.qzv" files[]=@"${PWD}/docs/moving-pictures/empress-tree-tandem.qzv" files[]=@"${PWD}/docs/moving-pictures/empress-tree-tandem-biplot.qzv" http://mchelper.ucsd.edu:8888/uploads/empress/${TRAVIS_PULL_REQUEST}/
```
Comment (ElDeveloper) [inline .travis.yml:38]:
```suggestion
  - curl -POST -F files[]=@"${PWD}/docs/moving-pictures/empress-tree.qzv" files[]=@"${PWD}/docs/moving-pictures/empire.qzv" files[]=@"${PWD}/docs/moving-pictures/empire-biplot.qzv" http://mchelper.ucsd.edu:8888/uploads/empress/${TRAVIS_PULL_REQUEST}/
```

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempress-tree.qzv)

**Comment by emperor-helper** [comment]
The following artifacts were built for this PR: [empire-biplot.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempire-biplot.qzv), [empire.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempire.qzv), [empress-tree.qzv](https://view.qiime2.org/visualization/?type=html&src=https%3A%2F%2Fmchelper.ucsd.edu%2Fdownloads%2Fempress%2F295%2Fempress-tree.qzv)

**Comment by ElDeveloper** [comment]
Thanks so much @antgoza

**Pull request merged**